packages/fast-components/src/badge/fixtures/base.html

```diff
@@ -0,0 +1,71 @@
+        Badge
+    </fast-badge>
+    <fast-badge fill="secondary" color="foo" id="foo" aria-label="Simple badge" modal="true">
+        Badge
+    </fast-badge>
+    <fast-badge fill="tertiary" style="font-size: 14px; line-height: 20px;" color="bar" id="foo" aria-label="Simple badge" modal="true">
+        Badge
+    </fast-badge>
+    <h4>Circular</h4>
+    <fast-badge fill="primary" color="foo" id="foo" circular="true" aria-label="Simple badge" modal="true">
+        9
+    </fast-badge>
+    <fast-badge fill="secondary" color="foo" id="foo" circular="true" aria-label="Simple badge" modal="true">
+        99
+    </fast-badge>
+    <fast-badge fill="tertiary" color="bar" id="foo" circular="true" aria-label="Simple badge" modal="true">
+        9999999
+    </fast-badge>
+    <h4>Examples</h4>
+    <div>
+        <div style="position: relative; display: inline-flex;">
+            <img src="https://placehold.it/120x140/DFDFDF/171717">
+            <fast-badge style="position: absolute; right: 4px; top: 4px;" fill="foo" color="foo" id="foo" aria-label="Simple badge" modal="true">
+                Sale
+            </fast-badge>
+        </div>
+        <div style="position: relative; display: inline-flex;">
+            <img src="https://placehold.it/140x140/DFDFDF/171717">
+            <fast-badge style="position: absolute; right: 4px; top: 4px;" fill="secondary" color="foo" id="foo" aria-label="Simple badge" modal="true">
+                App of the day
+            </fast-badge>
+        </div>
+    </div>
+    <div>
+        <div style="position: relative; display: inline-flex; width: 50px; height: 50px; background: #DFDFDF; margin: 12px;">
+            <fast-badge style="position: absolute; right: -6px; top: -6px;" circular="true" fill="bar" color="foo" id="foo" aria-label="Simple badge" modal="true">
+                3
+            </fast-badge>
+        </div>
+        <div style="position: relative; display: inline-flex; width: 50px; height: 50px; background: #DFDFDF; margin: 12px;">
+            <fast-badge style="position: absolute; right: -6px; top: -6px;" circular="true" fill="bar" color="foo" id="foo" aria-label="Simple badge" modal="true">
+                35000
+            </fast-badge>
+        </div>
+        <div style="position: relative; display: inline-flex; width: 50px; height: 50px; background: #DFDFDF; margin: 12px;">
+            <fast-badge style="position: absolute; right: -6px; top: -6px;" circular="true" fill="bar" color="foo" id="foo" aria-label="Simple badge" modal="true">
+                999
+            </fast-badge>
+        </div>
+        <div style="position: relative; display: inline-flex; width: 50px; height: 50px; background: #DFDFDF; margin: 12px;">
+            <fast-badge style="position: absolute; right: -6px; top: -6px;" circular="true" fill="bar" color="foo" id="foo" aria-label="Simple badge" modal="true">
+                15
+            </fast-badge>
+        </div>
+    </div>
```

Comment:
Without context I'm not fully understanding the scope of the examples here. If these are configurations to accomplish what was outlined in the spec it seems fairly complex to create different variations of badges.

Reply:
I'm just showing a badge overlaid on top of a another element. If you don't think it's relevant I can remove it.

![image](https://user-images.githubusercontent.com/37851214/78272091-e5bb7800-74c1-11ea-8bc3-83c238c8c8b7.png)